src/vivarium_csu_alzheimers/components/alzheimers.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
+from vivarium import Component
+from vivarium.framework.state_machine import State, TransientState
+from vivarium_public_health.disease import DiseaseModel, DiseaseState, SusceptibleState
+
+from vivarium_csu_alzheimers.constants.models import ALZHEIMERS_DISEASE_MODEL
+
+# Constants for the Alzheimer's disease model
+# These values are placeholders and should be replaced with actual data from research later
+PREVALENCE = 1 / 6
+EXCESS_MORTALITY_RATE = 0.01
```

Comment:
Question for research: Do we want excess mortality for the severe disability state only?